tests/by-util/test_tail.rs

```diff
@@ -2435,6 +2423,658 @@ fn test_illegal_seek() {
     );
     assert_eq!(p.wait().unwrap().code().unwrap(), 1);
 }
+// FIXME: Occasionally broken pipe on macos when no output is expected. Switch on such cases for macos when fixed.
+#[cfg(all(not(target_os = "android"), not(target_os = "windows")))] // FIXME: See https://github.com/uutils/coreutils/issues/3881
+mod pipe_tests {
+    use super::*;
+    use crate::common::random::*;
+    use rand::distributions::Alphanumeric;
+    const CHUNK_BUFFER_SIZE: usize = 8192;
```

Comment:
```suggestion
use tail::chunks::BLOCK_SIZE as CHUNK_BUFFER_SIZE;
```
together with `pub mod chunks;` in `tail.rs`

Reply:
Besides using BUFFER_SIZE instead of BLOCK_SIZE, I applied your changes